vendor/QmXvNwFNR7CQJ1J5oJnjS4oAFv2ai6v9JZBkTBbJp6wRQf/go-ipfs-gateway/gateway_handler.go

```diff
@@ -21,6 +21,8 @@ import (
 	path "github.com/ipfs/go-ipfs/path"
 	"github.com/ipfs/go-ipfs/routing"
 	uio "github.com/ipfs/go-ipfs/unixfs/io"
+
+	logging "github.com/rht/go-ipfs-gateway/vendor/QmQg1J6vikuXF9oDvm4wpdeAUvvkVEKW1EYDw9HhTMnP2b/go-log"
```

Comment:
Updating the pathing of 3 packages won't cause serious merge conflicts, however.